tools/jenkins/grpc_interop_php/Dockerfile

```diff
@@ -100,6 +100,10 @@ RUN /bin/bash -l -c "rvm all do gem install ronn rake"
 RUN curl -sS https://getcomposer.org/installer | php
 RUN mv composer.phar /usr/local/bin/composer
 
+# attempt to force a rebuild of the docker image after this point because
+# Protobuf-PHP codegen has been updated
+RUN echo 1
```

Comment:
yea, but I remove it, somehow Jenkins is smart enough to go back to the cached docker image and we are back to the old state. I will remove this as a separate PR and hopefully that will work.